.gitignore

```diff
@@ -1,5 +1,4 @@
-/target
-/local
```

Comment:
Any idea where does the /local come from?

Reply:
If I had to guess, it's a copy-paste error from the solver, where a `local` folder is where benchmarks store their results.